lib/Db/BoardMapper.php

```diff
@@ -42,7 +42,10 @@ class BoardMapper extends QBMapper implements IPermissionMapper {
 
 	private $circlesEnabled;
 
+	/** @var CappedMemoryCache */
```

Comment:
- [ ] reuse boardCache and only store board ids in the userBoardCache

Reply:
Thinking more about that it should be fine as it is, the current way has some memory overhead but we can avoid juggling around the existing objects to map to board ids and the other way round when obtaining the boards from the cache.